### PR TITLE
add configs for prod 20240716_v0.10.7_src10_dec3476_tuned

### PR DESCRIPTION
**Pull request by ToraTherese**
# New Prod config

Self check-list:

- [ ] I have checked the lstchain config, in particular for:
  - [ ] `az_tel` instead of `sin_az_tel` if data to be analyzed have been produced with lstchain <= v0.9.7
  - [ ] "increase_nsb" and "increase_psf" are provided in "image_modifier" (if used)
- [ ] I have checked the environment in the lstmcpipe config and it is the one used to analyse DL>1 data
- [ ] I have provided the command (in README), or script (in additionnal .py file) used to produce the lstmcpipe config

## Prod_ID

20240716_v0.10.7_src10_dec3476_tuned

## Short description of the config

Config for reprocessing of Gamma Diffuse MC at declination line of 34.76 deg, based on 20240430_v0.10.4_src9$
NSB tuning adapted to the observed data of src10.

## Why this config is needed 

Need diffuse MC with tuning for new source: src10

### CODE REVIEW

**Review by vuillaut — APPROVED** [review]
LGTM

**Comment by vuillaut** [comment]
> # New Prod config
> Self check-list:
> 
> * [ ]  I have checked the lstchain config, in particular for:
>   
>   * [ ]  `az_tel` instead of `sin_az_tel` if data to be analyzed have been produced with lstchain <= v0.9.7
>   * [ ]  "increase_nsb" and "increase_psf" are provided in "image_modifier" (if used)
> * [ ]  I have checked the environment in the lstmcpipe config and it is the one used to analyse DL>1 data
> * [ ]  I have provided the command (in README), or script (in additionnal .py file) used to produce the lstmcpipe config
> 
> ## Prod_ID
> 20240716_v0.10.7_src10_dec3476_tuned
> 
> ## Short description of the config
> Config for reprocessing of Gamma Diffuse MC at declination line of 34.76 deg, based on 20240430_v0.10.4_src9$ NSB tuning adapted to the observed data of src10.
> 
> ## Why this config is needed
> Need diffuse MC with tuning for new source: src10

Hi

Unfortunately this is failing because 
```
/fefs/aswg/data/mc/DL1/AllSky/20240430_v0.10.4_src9_dec3476_tuned/TrainingDataset/dec_3476/GammaDiffuse/
```
is empty 😬 

The dataset had been separated in train/test ones:
```
/fefs/aswg/data/mc/DL1/AllSky/20240430_v0.10.4_src9_dec3476_tuned/Split_TestDataset/
/fefs/aswg/data/mc/DL1/AllSky/20240430_v0.10.4_src9_dec3476_tuned/Split_TrainingDataset/
```

So you could start from these and do the dl1ab stage without the train/test split stage, e.g.:
```
input:  /fefs/aswg/data/mc/DL1/AllSky/20240430_v0.10.4_src9_dec3476_tuned/Split_TrainingDataset/dec_3476/GammaDiffuse/node_corsika_theta_12.829_az_301.263_/
output: /fefs/aswg/data/mc/DL1/AllSky/20240716_v0.10.7_src10_dec3476_tuned/Split_TrainingDataset/dec_3476/GammaDiffuse/node_corsika_theta_12.829_az_301.263_
```
etc... 

opening an issue...

File: production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+            "waveform_scale": 80
+        }
+    },
+    "image_modifier": {
+      "increase_nsb": true,
+      "extra_noise_in_dim_pixels": 1.327,
+      "extra_bias_in_dim_pixels": 0.41,
+      "transition_charge": 8,
+      "extra_noise_in_bright_pixels": 0.0
+    }
```
Comment (morcuended) [inline production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json:321]: this production seems quite similar to `20240430_v0.10.4_src9_dec3476_tuned` (same dec line and somewhat close image modifier parameter). @ToraTherese have you tried that production already (even if later more fine-tuning of the NSB is needed)?

Comment (ToraTherese) [reply production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json:321]: It is indeed very similar, just different image modifier parameter. What do you mean by using a later fine-tuning of the NSB?

Comment (vuillaut) [reply production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json:321]: Hi @ToraTherese 
I think @morcuended is actually right here.
The difference between production `20240430_v0.10.4_src9_dec3476_tuned` and the one you are requesting is 

```
"image_modifier": {
    "increase_nsb": true,
    "extra_noise_in_dim_pixels": 1.577,
    "extra_bias_in_dim_pixels": 0.463,
    "transition_charge": 8,
    "extra_noise_in_bright_pixels": 1.892
  }
```
vs
```
"image_modifier": {
      "increase_nsb": true,
      "extra_noise_in_dim_pixels": 1.327,
      "extra_bias_in_dim_pixels": 0.41,
      "transition_charge": 8,
      "extra_noise_in_bright_pixels": 0.0
    }
```

Using directly the trained model from `20240430_v0.10.4_src9_dec3476_tuned` for src10 would probably lead to very similar results - have you tried already?

Comment (vuillaut) [reply production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json:321]: As a reference, you may have a look at https://indico.cta-observatory.org/event/5532/contributions/45108/attachments/25665/37594/DL2_auto.pdf

The difference of nsb here (0.25 or 16%) is quite small and differences will most probably be hidden by systematics.

Comment (vuillaut) [reply production_configs/20240716_v0.10.7_src10_dec3476_tuned/lstchain_config_2024-07-16.json:321]: Hi @ToraTherese 
Have you been able to work with existing production `20240430_v0.10.4_src9_dec3476_tuned` ?
Should #473 be closed ?

**Pull request merged**